src/spatial/2d.js

```diff
@@ -81,6 +82,14 @@ Crafty.c("2D", {
      * @see ._attr
      */
     _z: 0,
+
+    /**@
+     * #._globalZ
+     * @comp 2D
+     * When two entities overlap, the one with the larger `_globalZ` will be on top of the other.
+     */
+    _globalZ: null,
```

Comment:
It's updated by changing `z`, though, so I'm inclined to leave them together.

Also, I think some of the other components *should* be using `_globalZ`.  Mouse dispatch compares on `z` and uses id as a tie breaker, but if it used `_globalZ` there'd never be any ties.